3_Automation/06_Bestiary.yml

```
@@ -0,0 +1,359 @@
+      Save:
+        Trigger: On hit
+        DR: 2
+        Type: STR                     
+        Fail: Knocked Down. 
+  Primary Element: Earth
+
+Shrub Golem:
+  Type: Bystander
+  HP: 2
+  AP: 0
+  AR: 4
+  Actions:
+    Attack, Weapon:
+      Type: Major
+      To hit: 1
+      Damage: 1            
+      Range: 0
+      Target: 1                 
+  Primary Element: Earth
+
+Vine Golem:
+  Type: Bystander
+  HP: 2
+  AP: 0
+  AR: 4
+  Skill Mods:
+    Str: 1
+    Agi: -1
+  Actions: 
+    Entangling Grasp:
+      Type: Major
+      To hit: 1
+      Range: 0
+      Target: 1
+      Save:
+        DR: 3
+        Type: STR
+        Fail: Entangle
+    Attack, Ranged:
+      Type: Major
+      To hit: 1
+      Damage: 1            
+      Range: 6
+      Target: 1       
+  Primary Element: Earth
+
+Tree Golem:
+  Type: Bystander
+  HP: 7
+  AP: 0
+  AR: 3
+  Skill Mods:
+    STR: 2
+    VIT: 2
+    AGL: -1
+    INT: -1
+  Actions:
+    Slam:
+      Type: Major
+      To hit: 2
+      Damage: 2          
+      Range: 0
+      AOE: Aura 1
+      Effect: The Golem slams the ground around it, attempting to knock nearby creatures down.
+      Save:
+        Trigger: On hit
+        DR: 2
+        Type: AGL                     
+        Fail: Knocked Down.
+        Succeed: Half damage, not Knocked Down.
+    Natural Regeneration:
+      Type: Minor
+      Effect: The Golem regenerates 1 HP. This can be used twice per Rest.
+  Primary Element: Earth
+
+Poison Plant:
+  Type: Bystander
+  HP: 2
+  AP: 0
+  AR: 4
+  Skill Mods:
+    AGL: -1
+    VIT: 1
+  Vulnys: 
+    Planted: The Plant has a Speed of 0 and cannot move on its own. 
+  Actions: 
+    Poison Spit:
+      Type: Major
+      Range: 6
+      Target: 1
+      To hit: 1
+      Damage: 1
+      Damage type: Poison
+    Entangling Grasp:
+      Range: 0
+      Target: 1
+      To hit: 1
+      Save:
+        DR: 3
+        Type: STR 
+        Fail: Entangle
+  Primary Element: Earth
+
+Grunt:
+  Type: Bystander
+  HP: 2
+  AP: 0
+  AR: 3
+  Skill Mods:
+    STR: 1
+  Actions:
+    Attack, Weapon:
+      Type: Major
+      To hit: 1
+      Damage: 1            
+      Range: 0
+      Target: 1       
+
+Marksman:
+  Type: Bystander
+  HP: 2
+  AP: 1
+  AR: 3
+  Skill Mods: 1
+
+Captain:
+  Type: Bystander
+  HP: 4
+  AP: 1
+  AR: 2
+  Skill Mods:
+    STR: 2
+    AGI: 1
+    CON: 1
+    INT: 1
+  Actions: 
+    Attack, Weapon:
+      Type: Major
+      To hit: 2
+      Damage: 1            
+      Range: 0
+      Target: 1
+    Command Grunt: 
+      Type: Minor
+      Effect: Targets one Grunt it can see. That Grunt can move up to 2 spaces and take a Major Action.
+    Raise Shield:
+      Type: Minor
+      Effect: The Captain gains 1 AP, up to a maximum of 1.      
+
+Ritualist:
+  Type: Bystander
+  HP: 2
+  AP: 0
+  AR: 3
+  PP: 2
+  Skill Mods:
+    INT: 2
+    STR: -1
+    AGI: 1
+    CON: 1
+  Actions: 
+    Attack, Mystic:
+      Type: Major
+      Range: 10
```

For players, this is a range6. Maybe to differentiate
```suggestion
    Attack, Mystic, Distant:
      Type: Major
      Range: 10
```

---

In a future version we will add the Passive for this creature enabling Distant casting to solve for this.